add rn entry for execution_project (#6631)
dbt-labs/docs.getdbt.com: website/docs/docs/dbt-versions/release-notes.md

## December 2024

> - **Fix**: The dbt Semantic Layer now respects the `execution_project` setting, including for exports.

Author: ```suggestion
- **Fix**: [The dbt Semantic Layer](/docs/use-dbt-semantic-layer/dbt-sl) now respects the `execution_project` setting, including for exports.
```

> - **Fix**: [The dbt Semantic Layer](/docs/use-dbt-semantic-layer/dbt-sl) now respects the `execution_project` setting, including for exports.

Reviewer: Looks good! I wonder if we should mention this is specifically for BigQuery?

Author: good shout thank you @aiguofer ! updated it now to this:

- **Fix**: [The dbt Semantic Layer](/docs/use-dbt-semantic-layer/dbt-sl) now respects the BigQuery [`execution_project` attribute](/docs/core/connect-data-platform/bigquery-setup#execution-project), including for exports.

> - **Fix**: [The dbt Semantic Layer](/docs/use-dbt-semantic-layer/dbt-sl) now respects the `execution_project` setting, including for exports.

Author: ```suggestion
- **Fix**: [The dbt Semantic Layer](/docs/use-dbt-semantic-layer/dbt-sl) now respects the BigQuery `execution_project` attribute setting, including for exports.
```

> - **Fix**: [The dbt Semantic Layer](/docs/use-dbt-semantic-layer/dbt-sl) now respects the BigQuery `execution_project` attribute setting, including for exports.

Author: ```suggestion
- **Fix**: [The dbt Semantic Layer](/docs/use-dbt-semantic-layer/dbt-sl) now respects the BigQuery [`execution_project` attribute](/docs/core/connect-data-platform/bigquery-setup#execution-project), including for exports.
```

Author: merging now!
